pulsar-functions/proto/src/main/proto/Function.proto

```diff
@@ -85,6 +85,7 @@ message SinkSpec {
 
 message PackageLocationMetaData {
     string packagePath = 1;
+    string packageUrl = 2;
```

Comment:
should we have another field here? or can we just use the existing field 'packagePath' and switch logic on the prefix e.g. http

Reply:
I wanted to avoid deriving logic which can create issue if url and path has conflicting convention in future. however, as @sijie mentioned if we want to deprecate jar-submission then we can just keep one field. fixed it.